tox.ini

```diff
@@ -40,6 +40,7 @@ description = Compile the requirements.txt file for docs
 deps = pip-tools
 commands =
     pip-compile --extra=docs -o docs/requirements.txt pyproject.toml
+    python -c 'open("docs/requirements.txt", "a").write("./testing/\n")'
```

Comment:
Interesting that testing is specified relative to the parent directory of docs, but I assume this is correct

Reply:
The actual `pip install` is done at the root level, so the path is relative to that. It does work locally (you can test with `tox -e docs`, optionally doing `tox -e docs-deps` first) and on rtd (you can look at the link from the passing CI step).